<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>softwareinfo</class>
 <widget class="QDialog" name="softwareinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>GIT_VERSION:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>BUILD_DATE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>COMPILER_VERSION:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_git_version">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_build_date">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_compiler_version">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_author">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Author: Giovanni Rasera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>email: g.rasera@huracanmarine.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string>github: GiovanniRaseraF</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_git_branch">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
font: 15pt &quot;Ubuntu Bold&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="candump_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>951</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Ubuntu Bold&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>candump</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>80</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/logotrasparent.png) ;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_author_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Ubuntu Bold&quot;;
color: rgb(28, 113, 216);</string>
   </property>
   <property name="text">
    <string>www.huracanmarine.com</string>
   </property>
  </widget>
  <zorder>l_logo</zorder>
  <zorder>buttonBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>l_git_version</zorder>
  <zorder>l_build_date</zorder>
  <zorder>l_compiler_version</zorder>
  <zorder>l_author</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>l_git_branch</zorder>
  <zorder>candump_text</zorder>
  <zorder>label_4</zorder>
  <zorder>l_author_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>softwareinfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>softwareinfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
